src/common/syncjournaldb.cpp

```diff
@@ -742,6 +747,58 @@ bool SyncJournalDb::updateMetadataTableStructure()
                 }
             }
             commitInternal(QStringLiteral("update database structure: add %1 column").arg(columnName));
+            return;
+        }
+
+        // the column already exists, check if default has been set
+        if (setDefault) {
+            SqlQuery query(_db);
+            const auto selectDefault = QStringLiteral("SELECT dflt_value FROM pragma_table_info('metadata') WHERE name = '%1';").arg(columnName);
+            query.prepare(selectDefault.toLatin1());
+            if (!query.exec()) {
+                sqlFail(QStringLiteral("check default value for: %1").arg(columnName), query);
+                re = false;
+            }
+
+            if (const auto result = query.next();result.ok && result.hasData && !query.nullValue(0)) {
+                qCWarning(lcDb) << "column" << columnName << "has default value of" << query.intValue(0);
```

Comment:
since this should be the more common case, change the info level to `Debug`
 
```suggestion
                qCDebug(lcDb) << "column" << columnName << "has default value of" << query.intValue(0);
```